config/locales/views.de.yml

```diff
@@ -51,6 +51,6 @@ de:
       contactable_by: Kontaktfreigabe
     fields_jubla:
       help_canton: Wird aus der PLZ berechnet falls leer.
-      info_alumnus_contactable: "Mit dem Setzen eines Häkchens erlaubst du folgenden Ebenen, dich zu kontaktieren und dir Informationen zu senden:"
+      info_alumnus_contactable: "Mit dem Setzen eines Häkchens erlaubst du folgenden Ebenen, dich zu kontaktieren und dir Informationen zu senden:"
```

Comment:
I have no idea why this also changed.

Reply:
Maybe an encoding thing, I think you can write an A Umlaut by one character or with a modifier ¨ and a.

Reply:
Yeah. That’s probably it.